cicd/Jenkins/Jenkinsfile

```diff
@@ -681,6 +690,29 @@ pipeline {
           }
         }
       }
+
+      stage('Create PR') {
+        agent {
+            dockerfile {
+              filename 'Dockerfile'
+              dir 'cicd/docker/autopr'
```

Comment:
this dockerfile does nothing but use an image; why not reference the image here direcly?

Reply:
I would prefer to do that, but we are getting errors when retrieving the Credentials `Could not find credentials matching docker-login-user-pass`
As example: https://jenkinsprod.corp.intranet:8443/blue/organizations/jenkins/UX-CHI%2FProductive%2FChi/detail/PR-239/93/pipeline/

This error seems to be known since those credentials were used, and then commented (on the other sections of the code)

I'll investigate if this happened for some reason that I don't understand (that is, the removal of the use of those credentials)